src/styleguide/sg-usedModules/usedModules.pug

```diff
@@ -0,0 +1,10 @@
+
+mixin sg-usedModules()
+  -
+    var data = self.usedModules;
```

Comment:
`data` is a very generic name for a variable, `usedModules` would be much better. Also, it should have a fallback value. (e.g. an empty array)